src/tool_getparam.c

```diff
@@ -1026,6 +1031,52 @@ static const struct LongShort *single(char letter)
   return singles[letter - ' '];
 }
 
+#ifdef IPTOS_DSCP_AF11
```

Comment:
This is a bit ugly. Do you prefer to add a configure check (for `IP_TOS`? something else?), check here for `IP_TOS` or just remove these abbreviations altogether?

Reply:
Replaced with `IP_TOS`. Looks like (at least on FreeBSD) `netinet/in.h` has to be included before `netinet/ip.h` anyway.

Let's see if it compiles on all platforms.

Reply:
Ok, no more macros. I just used the numeric values inline (they are invariant).